src/tracing/esp32_trace/include/matter/tracing/macros_impl.h

```diff
@@ -21,17 +21,100 @@
 #error "Tracing macros seem to be double defined"
 #endif
 
+#include <crypto/CHIPCryptoPAL.h>
+#include <lib/core/CHIPError.h>
+
 namespace Insights {
 class ESP32Backend
 {
 public:
     ESP32Backend(const char * str, ...);
     ~ESP32Backend();
 
+    static void AddStringToBlacklist(const char * aLabel)
```

Comment:
all methods in this class and all variables are static. Is there a reason they are in a class? Why not place methods in a namespace and put the variables and the private methods in the C++ file as an anonymous namespace?